src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -805,6 +805,24 @@
         - name: --defender-config
           type: string
           short-summary: Path to JSON file containing Microsoft Defender profile configurations.
+        - name: --enable-azuremonitormetrics
+          type: bool
+          short-summary: Enable Azure Monitor Metrics Profile
+        - name: --azure-monitor-workspace-resource-id
+          type: bool
+          short-summary: Resource ID  of the Azure Monitor Workspace
+        - name: --ksm-metriclabelsallowlist
```

Comment:
Maybe `--ksm-metric-labels-allow-list` is better?

Reply:
agreed, updated